hbase-server/src/main/java/org/apache/hadoop/hbase/replication/HBaseReplicationEndpoint.java

```diff
@@ -283,4 +291,8 @@ public AsyncRegionServerAdmin getRegionServer() {
       return regionServer;
     }
   }
+
+  public static void main(String[] args) {
```

Comment:
Do you still need this?

Reply:
Thanks for catching this. Let me remove, should be test code.